docs-v2/integrations/all/avalara-sandbox/connect.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Avalara(Sandbox) - How do I link my account?
+sidebarTitle: Avalara(Sandbox)
```

Comment:
```suggestion
sidebarTitle: Avalara (Sandbox)
```
applicable for this whole file

Reply:
🙏 updated as well with latest [PR](https://github.com/NangoHQ/nango/pull/3106/commits/e422a67d7de67e21ebe040ffe190dd9cab2d1663)